Самсонник Кирилл БИВТ-24-15 purple

Ahedoggo opened the pull request.

Ahedoggo committed:
Add files via upload

NemirovichMark commented:
Здравствуйте, Кирилл.
![image](https://github.com/user-attachments/assets/477236ca-c773-496e-a66b-4f511732df41)
Описание ошибок можно найти в словаре:
https://newlms.misis.ru/mod/glossary/view.php?id=65847&mode=letter&hook=SPECIAL
Выгрузка системы тестирования в файле (обращайте внимание на первую строку с подсказкой, где ошибка находится (должно содержать в пути часть: BIVT-2024-Lab-6-test\Lab_6\), а именно line №...:
[Lab_6_log_Purple.txt](https://github.com/user-attachments/files/19840372/Lab_6_log_Purple.txt)

Ahedoggo committed:
Add files via upload

NemirovichMark commented:
Здравствуйте, Кирилл.
[Lab_6_log_Purple.txt](https://github.com/user-attachments/files/19866197/Lab_6_log_Purple.txt)

Ahedoggo committed:
Add files via upload

NemirovichMark commented:
Здравствуйте, Кирилл.
Почти.
Описание ошибок можно найти в словаре (для 4го номера):
https://newlms.misis.ru/mod/glossary/view.php?id=65847&mode=letter&hook=SPECIAL
Для 5го номера внимательно читайте условие.
[Lab_6_log_Purple.txt](https://github.com/user-attachments/files/19889275/Lab_6_log_Purple.txt)

Ahedoggo committed:
Add files via upload

Марк Анатольевич, не подскажете, в чем может быть ошибка метода CountVotes в пятом задании? Не вижу, что там может работать не так

NemirovichMark commented:
Здравствуйте, Кирилл.
Ноуп
[Lab_6_log_Purple.txt](https://github.com/user-attachments/files/19973807/Lab_6_log_Purple.txt)

Ahedoggo commented:
💀
